imagestore/internal/securemountapi/imagestore.proto

```diff
@@ -1,7 +1,7 @@
 // source: https://github.com/kata-containers/kata-containers/blob/997a1f35ab68ad8cc2b24e8e20ea63f7de82acf9/src/libs/protocols/protos/confidential_data_hub.proto
 syntax = "proto3";
 
-package securemountapi;
+package api;
```

Comment:
Maybe leave a note here not to rename it?

Reply:
Sure